src/ripple/app/misc/TxQ.h

```diff
@@ -405,41 +608,68 @@ class TxQ
 
     using AccountMap = std::map <AccountID, TxQAccount>;
 
+    /// Setup parameters used to control the behavior of the queue
     Setup const setup_;
+    /// Journal
     beast::Journal j_;
 
-    // These members must always and only be accessed under
-    // locked mutex_
+    /** Tracks the current state of the queue.
+        @note This member must always and only be accessed under
+        locked mutex_
+    */
     FeeMetrics feeMetrics_;
+    /** The queue itself: the collection of transactions ordered
+        by fee level.
+        @note This member must always and only be accessed under
+        locked mutex_
+    */
     FeeMultiSet byFee_;
+    /** All of the accounts which currently have any transactions
+        in the queue. Entries are created and destroyed dynamically
+        as transactions are added and removed.
+        @note This member must always and only be accessed under
+        locked mutex_
+    */
     AccountMap byAccount_;
+    /** Maximum number of transactions allowed in the queue based
+        on the current metrics. If uninitialized, there is not limit,
+        but that condition can not last for long in practice.
+        @note This member must always and only be accessed under
+        locked mutex_
```

Comment:
Two suggestions:
- "... there is _no_ limit ..."
- "... condition _cannot_ last ..."

Reply:
Fixed